internal/generate/run.go

```diff
@@ -16,13 +16,37 @@ package generate
 
 import (
 	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/googleapis/generator/internal/gitrepo"
 )
 
 func Run(ctx context.Context, arg ...string) error {
 	cfg := &config{}
-	_, err := parseFlags(cfg, arg)
+	cfg, err := parseFlags(cfg, arg)
 	if err != nil {
 		return err
 	}
+	if _, err := cloneGoogleapis(ctx); err != nil {
+		return err
+	}
+	if _, err := cloneLanguageRepo(ctx, cfg.language); err != nil {
+		return err
+	}
 	return nil
 }
+
+const googleapisURL = "https://github.com/googleapis/googleapis"
+
+func cloneGoogleapis(ctx context.Context) (*gitrepo.Repo, error) {
+	repoPath := filepath.Join(os.TempDir(), "/generator-googleapis")
+	return gitrepo.CloneOrOpen(ctx, repoPath, googleapisURL)
+}
+
+func cloneLanguageRepo(ctx context.Context, language string) (*gitrepo.Repo, error) {
+	languageRepoURL := fmt.Sprintf("https://github.com/googleapis/google-cloud-%s", language)
+	repoPath := filepath.Join(os.TempDir(), fmt.Sprintf("/google-cloud-%s", language))
```

Comment:
Maybe `/generator-google-cloud-%s` to be consistent with `generator-googleapis`?

Reply:
Done